Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM circleci/openjdk:9
-RUN sudo apt-get update && sudo apt-get install gettext docker python-pip python-setuptools wget curl apt-transport-https
+FROM circleci/openjdk:11
```

Comment:
is there no corretto image we can just pull?

Reply:
No, unfortunately CircleCI does not provide one for Corretto: https://hub.docker.com/u/circleci

Reply:
@tkaschwig Yes, an image for Amazon Corretto is available but for CircleCI pipeline we need some extra setup. It's easier to use one of the provided CircleCI images and add Corretto JDK.

Reply:
@matthiaskeysermann-signavio Have you tried what is described here?
https://circleci.com/docs/2.0/custom-images/

Reply:
Thanks, will check that.